Events Page Redesign

oyenuga17 opened the pull request:
@sabine  please help me put on staging.

oyenuga17 committed:
events redesign

sabine committed:
fmt

sabine reviewed (commented):
![image](https://github.com/user-attachments/assets/0b9cb408-c33d-4240-ac5e-23954df9b388)

It would make sense to list only those locations that are actually there in the upcoming events.

Would also consider giving the URL query parameters more descriptive names.

oyenuga17 commented:
> ![image](https://private-user-images.githubusercontent.com/6594573/355033842-0b9cb408-c33d-4240-ac5e-23954df9b388.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI4NDc2NTcsIm5iZiI6MTcyMjg0NzM1NywicGF0aCI6Ii82NTk0NTczLzM1NTAzMzg0Mi0wYjljYjQwOC1jMzNkLTQyNDAtYWM1ZS0yMzk1NGRmOWIzODgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDgwNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA4MDVUMDg0MjM3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MDU0NjI4NjhjMDJiNTVmOWJmYzZhYmRmZGE0NWJkNWZkMzk2ODZkNTY3MTYzOWQzMDdkNzZkMzFlMDUwOGFlOSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.5WST5qyUT1aTD6mmsMjUX9ChZ6pQvCR1s_ql_F5rQPI)
> 
> It would make sense to list only those locations that are actually there in the upcoming events.
> True that makes sense i will look into this.

> Would also consider giving the URL query parameters more descriptive names.
Okay. This was what I did initially but I thought the parameters were too long so I shortened to include just the letters

oyenuga17 committed:
implement feedbacks:
- make url parameters more descriptive
- filter upcoming locations to only show available locations for upcoming events

sabine committed:
minor renaming

sabine reviewed (approved):
Merging, thanks @oyenuga17! :sparkles:
